recipes-kernel/linux/linux-sunxi_3.4.bb

```diff
@@ -3,10 +3,11 @@ require linux.inc
 DESCRIPTION = "Linux kernel for Allwinner a10/a20 processors"
 
 COMPATIBLE_MACHINE = "(sun4i|sun5i|sun7i)"
+DEPENDS = "sunxi-board-fex"
```

Comment:
In that case the end result is the same, but it might be better to use RDEPENDS, but that depends ... :-) Can change that if necessary